src/platform/toeplitz.c

```diff
@@ -75,7 +75,7 @@ CxPlatToeplitzHashInitialize(
     //
     // Initialize the Toeplitz->LookupTables.
     //
-    for (uint32_t i = 0; i < CXPLAT_TOEPLITZ_LOOKUP_TABLE_COUNT; i++) {
+    for (uint32_t i = 0; i < Toeplitz->InputSize; i++) {
```

Comment:
No `* NIBBLES_PER_BYTE` needed here?

Reply:
Thank you! I must have overlooked that during the conversion.